examples/chart/teleport-cluster/templates/serviceaccount.yaml

```diff
@@ -1,9 +1,12 @@
+{{- if ne false ((.Values.serviceAccount).create) -}}
```

Comment:
```suggestion
{{- if .Values.serviceAccount.create -}}
```

Reply:
This is a potentially breakable change, as after chart upgrade, for those who will use old values, this will raise a nullable error, as the `serviceAccount` field will not be defined. If that's not a concern, then all is good.